pytransform3d/transform_manager/_transform_manager.py

```diff
@@ -344,8 +344,8 @@ def set_transform_manager_state(self, tm_dict):
         self.nodes = tm_dict.get("nodes")
         self.i = tm_dict.get("i")
         self.j = tm_dict.get("j")
-        self.transform_to_ij_index = {
-            (tuple(k), v) for k, v in tm_dict.get("transform_to_ij_index")}
+        self.transform_to_ij_index = dict(
```

Comment:
the other way to fix it would be `{tuple(k): v for ...}`, I'm not sure what the difference is

Reply:
Yes that would be more easy to read. The result is the same. Even `dict(tm_dict.get("transform_to_ij_index"))` works, as it automatically converts a list to a dict. 

I just reverted the change that was done here https://github.com/dfki-ric/pytransform3d/commit/e403e29b4920fa003527ae344b3605cead90f548 to be on the save side (as this was the version that worked for me).